homeassistant/components/sql/strings.json

```diff
@@ -0,0 +1,41 @@
+{
+    "config": {
+        "abort": {
+            "already_configured": "[%key:common::config_flow::abort::already_configured_account%]"
+        },
+        "error": {
+            "db_url_invalid": "Database URL invalid",
+            "query_invalid": "SQL Query invalid",
+            "value_template_invalid": "Value Template invalid"
+        },
+        "step": {
+            "user": {
+                "data": {
+                    "db_url": "Database URL",
+                    "query": "Select Query",
+                    "column": "Column",
+                    "unit_of_measurement": "Unit of Measure",
+                    "value_template": "Value Template"
+                }
+            }
+        }
+    },
+    "options": {
+        "step": {
+            "init": {
+                "data": {
+                    "db_url": "Database URL",
```

Comment:
We should make these references to the other instances of this key. See the example in homekit/strings.json